build/README.md

```diff
@@ -1,57 +1,75 @@
-# kprobe build setup
+# Building kprobe
 
-kprobe is distributed as a static binary linked with musl libc which
-requires a proper cross compilation environment. A reproducible
-build environment can be created using docker:
+Building kprobe is honestly a mess at this point. The streamlined build
+processes have not been maintained in years so the required docker images
+can no longer be created. The previous build instructions can be found at
+the bottom of this document but are not recommended nor guaranteed to work.
 
-```
-docker build -t kprobe-build -f build/Dockerfile .
-docker run --rm -ti -v $PWD:/work -v $HOME/.ssh:/root/.ssh kprobe-build bash
-root@d798d406bab1:/work# cargo build --release --target x86_64-unknown-linux-musl
-```
+# Build Setup
 
-Public releases include .deb and .rpm packages which can be built
-inside the same docker container:
+Kprobe is built using the Rust toolchain. The current know working path to
+success assumes that builds are done from a recent version of macOS without
```

Comment:
question: there's no way to build this w/o a Mac?

Reply:
There is a way and previously, the build scripts supported it. Unfortunately, those scripts and the docker images haven't been maintained and no longer work. We don't have enough pressure (ie: we don't see enough benefit) to invest in making this process better at this time. 

Building outside of macOS requires a more advanced level of build tooling than the steps outlined above but is not impossible.